### ISSUE TITLE
Preview images for early multicellular species need to be implemented

### ISSUE THREAD

**Issue by stoptakingusernames** [Revolutionary-Games/Thrive#3899]
Portraits for multicellular species are not displayed, they can not be fossilized or viewed in the evolutionary tree

**To Reproduce**
1. Become multicellular
2. View your species in the evolutionary window

**Screenshots**

![evwindow](https://user-images.githubusercontent.com/33234241/205392026-ff99441a-910e-4cf3-bf32-bb67979da938.png)

**Additional context**
The "Stage" stat (which is stage: microbe basically every cell) also doesn't display on the specie info tab

**Comment by hhyyrylainen** [comment]
Renamed this issue a bit as it's entirely intentional in the game code that multicellular species are not added to the logic to process the evolutionary tree.

**Comment by Oliveriver** [comment]
Yeah, for now that’s intentional. It will be fixed once the general approach to the Multicellular Stage and its data structures are solidified. It could be worth displaying a note in this situation saying that.

**Comment by hhyyrylainen** [comment]
I think adding such a note would be much more effort than generating (badly cropped) images of the early multicellular species.
